### PR TITLE
[WIP] Migrate: FormTextField

### PR DESCRIPTION
**Pull request by 3lvis**
- Initial progress to remove agnostic dependencies out of Form

### CODE REVIEW

**Comment by 3lvis** [comment]
This will potentially remove some functionality from Form. This functionality could be introduced again afterwards using a more "swifty" approach.

**Comment by 3lvis** [comment]
Needs migrating the InputValidator class generator

File: Source/Cells/Text/FORMTextFieldCell.m [hunk 1]
```diff
@@ -76,11 +74,11 @@ - (void)dealloc {
...
 #pragma mark - Getters
 
-- (FORMTextField *)textField {
+- (FormTextField *)textField {
     if (_textField) return _textField;
 
-    _textField = [[FORMTextField alloc] initWithFrame:[self textFieldFrame]];
-    _textField.textFieldDelegate = self;
+    _textField = [[FormTextField alloc] initWithFrame:[self textFieldFrame]];
+    //_textField.textFieldDelegate = self;
```
Comment (zenangst) [inline Source/Cells/Text/FORMTextFieldCell.m]: wat?

Comment (3lvis) [inline Source/Cells/Text/FORMTextFieldCell.m]: ?

**Pull request closed without merging**